docs/source/intro.rst

```diff
@@ -1,24 +1,66 @@
 .. include:: ./links.inc
+.. _intro:
 
 Introduction
-------------
+============
+
+The Open Science Framework
+--------------------------
+
+The Open Science Framework (OSF_), developed and maintained by the Center for Open Science (COS), is a tool that promotes open, centralized workflows by enabling capture of different aspects and products of the research lifecycle, including developing a research idea, designing a study, storing and analyzing collected data, and writing and publishing reports or papers.
+In the scientific community, it is commonly used for registered reports, as a preprint server, and for study archival and data sharing.
+In order to use the OSF, a free registration is required.
+
+The core functionality of the OSF_ is its ability to create and develop *projects*, with a project being a private or public workspace for collaboration, data sharing, or data archival.
+Projects have an associate storage (either via OSF storage or third party providers) for (large) data, and can have *components*, associated sub-projects.
+Each OSF user, project, component, and file is given a unique, persistent uniform resource locator (URL) to enable sharing and promote attribution.
+Projects can also be assigned digital object identifiers (DOIs) and archival resource keys (ARKs) if they are made publicly available.
+
+At the moment, a the OSF storage provides virtually unlimited storage capacity.
+As long as individual files are smaller than 5GB, any amount of data can be uploaded to the OSF.
+This makes the OSF_ a powerful, accessible, and free data sharing and collaboration platform for researchers.
+
 
 Goal of the extension
-^^^^^^^^^^^^^^^^^^^^^
+---------------------
 
-This extension aims to allow DataLad_ to work with the Open Science Framework (OSF_).
-This is done by transforming storage on the Open Science Framework (OSF) into a `git-annex`_ repository.
+This extension allows DataLad_ to work with the Open Science Framework (OSF_) to make sharing and collaboration on data or DataLad datasets even easier.
+It comes with several features that enable the following main use cases:
+
+#. Export existing datasets to the OSF
+#. Clone published datasets from the OSF
```

Comment:
Is this possible right now?

Reply:
yes

Reply:
yes. you could give "usecase 1" a test run